internal/value/value.go

```diff
@@ -560,6 +635,22 @@ type doubleValue struct {
 	value float64
 }
 
+func (v *doubleValue) CastTo(dst interface{}) error {
+	switch vv := dst.(type) {
+	case *string:
+		*vv = strconv.FormatFloat(v.value, 'f', -1, 64)
+		return nil
+	case *[]byte:
```

Comment:
I suggent not cast any types (but Bytes) to []byte, because it can be used as binary (compact) representation of type.